lightning/src/chain/channelmonitor.rs

```diff
@@ -620,7 +620,7 @@ pub(crate) struct ChannelMonitorImpl<Signer: Sign> {
 	funding_redeemscript: Script,
 	channel_value_satoshis: u64,
 	// first is the idx of the first of the two revocation points
-	their_cur_revocation_points: Option<(u64, PublicKey, Option<PublicKey>)>,
+	their_cur_per_commitment_points: Option<(u64, PublicKey, Option<PublicKey>)>,
```

Comment:
I feel like variable names are free, so we could spell out "current" or "latest"

Reply:
nope, just marginally easier on the reader